src/export_data.py

```diff
@@ -0,0 +1,122 @@
+"""Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import argparse
+import ee
+from utils import TaskManager, SampleExporter, stack_bands_from_imagery
+from utils import RoadImagery, MultiSpectralImagery, TropomiImagery
+from utils import DSMImagery, WindImagery
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Export Data from EarthEngine'
+                                     )
+    parser.add_argument('--params_path', type=str,
+                        help="json parameters file location")
+    parser.add_argument('--export_folder', type=str,
+                        help="Cloud export folder")
+    parser.add_argument('--bucket', type=str,
+                        help="bucket name",
+                        default='aghriss-air-quality')
+    parser.add_argument('--num_samples', type=int,
+                        help="number of samples to export per image",
+                        default=1000)
+    parser.add_argument('--num_shards', type=int,
+                        help="number of sharding of samples before export",
+                        default=10)
+
+    args = parser.parse_args()
+
+    params = json.load(open(args.params_path, 'r'))
+    BUCKET = args.bucket
```

Comment:
I'm actually not sure that you want any global constants in your file.  Usually Global Constants are declared at the top and reused throughout the file (like different functions). But you just have a single main statement, so these variables are pretty much just normal variables.

Just found this in the styleguide, so let's just turn all of these global variables into normal variables:
https://google.github.io/styleguide/pyguide.html#25-global-variables